css/style.css

```diff
@@ -1,19 +1,219 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+ body{
```

Comment:
Be careful here that you try to style your code consistently - there's some good information on how to style available here: https://www.w3schools.com/html/html5_syntax.asp